cobbler/utils.py

```diff
@@ -91,12 +91,18 @@ def _(foo):
 
 
 def die(logger, msg):
+    """
+    This method arifically let's Cobbler crash with an exception. Log the exception once in the per-task log or the main
```

Comment:
arificially -> wrong spelled, better remove the word totally.

Below: Remove unfilled docstrings